src/olympia/addons/templates/addons/replacement_addons.html

```diff
@@ -0,0 +1 @@
+{% extends "impala/base_shared.html" %}
```

Comment:
I just copied it from other templates :)  `based_shared` seems to be generic template for when Marketplace was in the same codebase - I'll change it to `base`.